### PR TITLE
[#1459] do not merge in case .rultor file is in PR

### PR DESCRIPTION
**Pull request by pnatashap**
- logic is added 
- test is disabled, as MkPull doesn't allow to work with files (too many code to write without it)

### CODE REVIEW

**Comment by pnatashap** [comment]
@yegor256 please take a look

File: src/main/java/com/rultor/agents/github/qtn/QnMerge.java [hunk 1]
```diff
@@ -77,25 +76,34 @@ public Req understand(final Comment.Smart comment,
...
+                        false,
+                        QnMerge.PHRASES
+                                .getString("QnMerge.system-files-affected")
                 );
                 req = Req.DONE;
-            }
+            } else
+                if (pull.allChecksSuccessful()) {
+                    new Answer(comment).post(
+                            true,
```
Comment (yegor256) [inline src/main/java/com/rultor/agents/github/qtn/QnMerge.java:90]: @pnatashap indentation here is definitely wrong (not 4 spaces), but I can't understand why Qulice is not complaining about this...

File: src/main/resources/phrases_en_US.properties [hunk 2]
```diff
@@ -36,6 +36,8 @@ QnMerge.checks-are-failed=Can't merge it. Some CI checks were failed. \
                           Apparently, the pull request is not ready to be \
                           merged since it has some problems. \
                           Please, fix them first.
+QnMerge.system-files-affected=Can't merge it. File `.rultor.yml` is affected by \
+                              this pull request.
```
Comment (yegor256) [inline src/main/resources/phrases_en_US.properties:40]: @pnatashap would be great if we can explain to the user, why we don't accept modifications to this particular file. Let's say: "Can't merge this pull requests, because the file `.rultor.yml` was modified. This is a security threat. Make a separate pull request with the modification of this particular file and ask project architect to merge it manually."

Comment (yegor256) [reply src/main/resources/phrases_en_US.properties:40]: @pnatashap also, I think it's better not to encode the name of the file here, but use `%s` and pass the name of it as a parameter to `String.format`

File: src/test/java/com/rultor/agents/github/qtn/QnMergeTest.java [hunk 3]
```diff
@@ -186,6 +191,48 @@ public void continuesBecauseCiChecksSuccessful()
...
+        MatcherAssert.assertThat(
+                new Comment.Smart(this.comments.get(1)).body(),
+                Matchers.is(QnMergeTest.COMMAND)
+        );
+        MatcherAssert.assertThat(
+                new Comment.Smart(this.comments.get(2)).body(),
+                Matchers.containsString(
+                        QnMergeTest.PHRASES.getString(
+                            "QnMerge.system-files-affected"
+                        )
```
Comment (yegor256) [inline src/test/java/com/rultor/agents/github/qtn/QnMergeTest.java:231]: @pnatashap indentations here are also unusual, but Qulice doesn't complain... very strange.

**Comment by yegor256** [comment]
@pnatashap many thanks for the contribution! see my comments above

**Comment by pnatashap** [comment]
@yegor256 please take a look.
qulice does not complains most probably because the version here is 0.13, while the last one is 0.22 and there are 1000+ changes required. It is nice to reformat it, because some of the requirements are not compatible - in this author and version is required and it is prohibited now. May be also good to have public IntelliJ settings file, because these indentations are just default.

**Comment by yegor256** [comment]
@rultor merge
